end_to_end/tpu/deepseek/Run_DeepSeek.md

```diff
@@ -135,6 +135,47 @@ python3 -m MaxText.decode MaxText/configs/base.yml \
     scan_layers=False
 ```
 
+## Correctness
+To verify the correctness of the model implementation, we perform two primary checks:
+
+* **Logit Comparison**: We compare the logits generated by our implementation against those from a HuggingFace implementation for a set of given prompts.
+* **MMLU Score Validation**: We validate the MMLU score against established benchmarks.
+
+One example command to generate golden logits from HuggingFace for V2-Lite on CPU.
+
+```sh
+LOCAL_DIRECTORY='MaxText/test_assets/'
+python3 -m MaxText.scratch_code.generate_hf_golden_logits \
+    --model-id=deepseek-ai/DeepSeek-V2-Lite \
+    --output-path=golden_DeepSeek-V2-Lite.jsonl \
+    --prompts='I love to,Today is a,What is the,' \
+    --gcs_bucket=${LOCAL_DIRECTORY}
+```
+
+One example command to run logits comparison with V2-Lite on CPU.
+
+```sh
+python3 -m MaxText.tests.forward_pass_logit_checker \
```

Comment:
Just for my understanding - this will also be run with every PR as part of the unit tests, right?

Reply:
No, it's not in the unit test yet for this logit test, and same for other models.